src/vs/platform/encryption/electron-main/encryptionMainService.ts

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { createDecorator } from 'vs/platform/instantiation/common/instantiation';
+import { ICommonEncryptionService } from 'vs/platform/encryption/electron-main/common/encryptionService';
+import { encrypt, decrypt } from 'vscode-encrypt';
```

Comment:
@RMacfarlane to not block startup can you do an `await import('vscode-encrypt')` when needed? Any module we add to the startup will make startup slower.

Reply:
I am seeing `await require('vscode-encrypt')` but I think it should be `await import('vscode-encrypt')`